### ISSUE TITLE
CSS Configuration has no `authorityEnable` and `adminPwd` 

### ISSUE THREAD

**Issue by DanielHabenicht** [opentelekomcloud/terraform-provider-opentelekomcloud#889]
# Terraform Version
```
Terraform v0.14.7
+ provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.23.2
```

# Affected Resource(s)
 - opentelekomcloud_css_cluster_v1.css_cluster

# Terraform Configuration Files
```hcl
# env.tfvars
css_vpc_cidr          = "172.16.0.0/24"
css_subnet_cidr       = "172.16.0.0/24"
css_subnet_gateway_ip = "172.16.0.1"
css_node_number       = 1
css_flavor            = "css.medium.8"
css_volume_type       = "COMMON"
css_volume_size       = 40
css_availability_zone = "eu-de-01"

# main.tf
resource "opentelekomcloud_vpc_v1" "css_vpc" {
  name = "indicarus-css-vpc"
  cidr = var.css_vpc_cidr

  tags = {
    terraform = "true"
  }
}

resource "opentelekomcloud_vpc_subnet_v1" "css_subnet" {
  name       = "indicarus-css-subnet"
  cidr       = var.css_subnet_cidr
  vpc_id     = opentelekomcloud_vpc_v1.css_vpc.id

  gateway_ip = var.css_subnet_gateway_ip

  tags = {
    terraform = "true"
  }
}

resource "opentelekomcloud_networking_secgroup_v2" "css_sg" {
  name        = "indicarus-css-sg"
  description = "Security group for a CSS cluster."
}

resource "opentelekomcloud_networking_secgroup_rule_v2" "css_sg_rule" {
  direction         = "ingress"
  ethertype         = "IPv4"
  protocol          = "tcp"
  port_range_min    = 9200
  port_range_max    = 9200
  remote_ip_prefix  = "0.0.0.0/0"
  security_group_id = opentelekomcloud_networking_secgroup_v2.css_sg.id
}

resource "opentelekomcloud_css_cluster_v1" "css_cluster" {
  name            = "indicarus-css-cluster"
  expect_node_num = var.css_node_number
  node_config {
    flavor = var.css_flavor
    network_info {
      security_group_id = opentelekomcloud_networking_secgroup_v2.css_sg.id
      network_id        = opentelekomcloud_vpc_subnet_v1.css_subnet.id
      vpc_id            = opentelekomcloud_vpc_v1.css_vpc.id
    }
    volume {
      volume_type = var.css_volume_type
      size        = var.css_volume_size
    }

    availability_zone = var.css_availability_zone
  }
}
```

# Debug Output/Panic Output
Not applicable

# Steps to Reproduce
```
terraform init
terraform plan -out=tfplan
terraform apply tfplan
```

# Expected Behavior

There is no way to retrieve or set the Authorization options like `adminPwd`. 
This means there is no way to automate further actions, without manually going to the Web Portal. 

# Actual Behavior
No options supplied.

# Important Factoids
None

# References
Out of discussion in #878

**Comment by aengelhorn** [comment]
The documentation structure is misleading: one could read it in a way that the *enable_https* and *admin_pass* parameters belong to the *volume* block, but they are parameters of *opentelekomcloud_css_cluster_v1* itself.

**Comment by outcatcher** [comment]
@aengelhorn This is already fixed in #926, will be part of the `v1.23.6` release
